sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -196,6 +196,28 @@ The Azure Monitor Distro uses EventSource for its own internal logging. The logs
 OpenTelemetry also provides it's own [self-diagnostics feature](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/README.md#troubleshooting) to collect internal logs.
 An example of this is available in our demo project [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/OTEL_DIAGNOSTICS.json).
 
+**Missing Request Telemetry**
+
+If an app has a reference to the `OpenTelemetry.Instrumentation.AspNetCore` package, it could be missing request telemetry. To resolve this issue:
+
+* Either remove the reference to the `OpenTelemetry.Instrumentation.AspNetCore` package (or)
+* Add `AddAspNetCoreInstrumentation` to the OpenTelemetry TracerProvider configuration as per the [OpenTelemetry documentation](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore).
+
+**Few or all Dependency Telemetries are missing**
+
+If an app references the `OpenTelemetry.Instrumentation.Http` or `OpenTelemetry.Instrumentation.SqlClient` packages, it might be missing dependency telemetry. To resolve:
+
+* Remove the respective package references (or)
+* Add `AddHttpClientInstrumentation` or `AddSqlClientInstrumentation` to the TracerProvider configuration. Detailed guidance can be found in the OpenTelemetry documentation for [HTTP](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.Http) and [SQL Client](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.SqlClient).
+
+**Missing Application Insights Standard Metrics**
```

Comment:
Rethinking on it - I think we should remove this section. Standard metrics are always available. The only difference in this case would be how they are calculated (At ingestion versus in-process).

Reply:
Or add a note common to above two options - missing traces could result missing standard metrics.

Reply:
Agree. Will update the doc.

Reply:
> Or add a note common to above two options -> missing traces could result missing standard metrics.

This will make it difficult to understand, will keep it simple by removing it.